### DISCUSSION TITLE
Start date and end date are showing with past dates, "Next" button is not aligned properly and mandatory asterisk are not proper in the Conversation setup page

### DISCUSSION THREAD

**Discussion by ashokreddy1208** [samagra-comms/community#71] [General Discussion]
@abhay-samagra @aayushkedia95 @SaketSinha @chinmoy12c @karntrehan

**Steps:**
- Open the sunbird ED staging env
- Login with orgadmin credential
- Click on profile icon
- Click on communication console
- Click on Add New button
- Fill the mandatory fields 
- update the start and end date then click on next button
- Click on back button
- try to update the end date and observe

**Actual result :** User is able to click on yesterday date for start date
                      User is able to update end date lesser than start date
                      Next button and mandatory asterisk icon are not aligned properly 

**Expected result :** User should be able to click on current date for start date
                      User should be able to update end date greater than start date
                      Next button and mandatory asterisk icon should be  aligned properly

https://github.com/samagra-comms/community/assets/58033030/b299a4a9-29cb-4f4a-8d15-66f80bfb4f21

**Comment by ashokreddy1208** [comment]
@abhay-samagra @aayushkedia95 @SaketSinha @chinmoy12c @karntrehan Can you please look into this issues

**Comment by abhay-samagra** [comment]
Hi Chinmoy, Karan

Could you please have a look at this?

Thanks and Regards

Abhay Singh

Senior Product Consultant | +91 787 477 4342

Samagra | Transforming Governance

Address: 9/5 Sarvapriya Vihar, New Delhi-16
***@***.***,77.1994501,17z/data=!3m1!4b1!4m5!3m4!1s0x390ce273981e5dfb:0x7bfceb198af17562!8m2!3d28.5566128!4d77.2016388?shorturl=1>

Website: www.samagragovernance.in

Follow Us: LinkedIn
<https://www.linkedin.com/company/samagra-transforming-governance/> |
YouTube <https://www.youtube.com/channel/UCfkXErS-f87xUQkmSKSC8bg> |
Instagram <https://www.instagram.com/samagragovernance/?hl=en>

**Comment by ashokreddy1208** [comment]
@abhay-samagra, Yesterday we had a discussion with Abhishek Kumar Pal regarding this issue and shared required details, can you please share ETA for this issue.

**Comment by ashokreddy1208** [comment]
@abhay-samagra  is this issue resolved? can you please confirm and update this status on this issue

**Comment by ashokreddy1208** [comment]
@abhay-samagra @aayushkedia95 @SaketSinha @chinmoy12c @karntrehan

Can you please look into this issue on priority and provide fix.

**Comment by abhay-samagra** [comment]
Yes, Ashok we are checking internally.Sent from my iPhoneOn 26-Sep-2023, at 9:20 AM, ashokreddy1208 ***@***.***> wrote:﻿
@abhay-samagra @aayushkedia95 @SaketSinha @chinmoy12c @karntrehan
Can you please look into this issue on priority and provide fix.

**Comment by abhishekpnt** [comment]
@abhay-samagra Please review and merge this PR for the alignment issue fix

https://github.com/samagra-comms/uci-console/pull/39/files

CC: @ashokreddy1208 @rajeevsathish
